README.md

```diff
@@ -0,0 +1,21 @@
+# For common users:
+Set the needed variables in .env (take librephotos.env as model)
+Just pull the repo and run
+docker-compose up -d
+This will get the pre-built images and start all the needed processes
+
+# For developers:
+Set the needed variables in .evn (take librephotos.env as model)
+Also set the codedir variable that tells docker where your source folder are
+Pull frontend code into ${codedir}/frontend/
```

Comment:
Same here, we can give the commands to clone the repos in the different subdirectories.

Reply:
Absolutely. I added the repos but until pull requests are not accepted it will not work obviosuly